qa/suites/rbd/migration-external/8-migrate-workloads/execute.yaml

```diff
@@ -0,0 +1,12 @@
+tasks:
+  - parallel:
+      - io_workload
+      - migrate_workload
+migrate_workload:
+  sequential:
+    - exec:
+        cluster1.client.0:
+          - sleep 60
+          - rbd --cluster cluster1 migration execute client.0.0
+          - sleep 60
+          - rbd --cluster cluster1 migration commit client.0.0
```

Comment:
The existing migration snippet doesn't do anything with `client.0.2`:
https://github.com/ceph/ceph/blob/64f5bff3791df7656457f2462cf8b6aeb824549f/qa/suites/rbd/migration/8-migrate-workloads/execute.yaml#L8-L14

I assume that since there are 3 images, the idea was to do both `rbd migration execute` and `rbd migration commit` on the first one, only `rbd migration execute` on the second one and nothing with the third one to cover all possible combinations.  But I could be wrong and just imagining things -- I wasn't involved with `qa/suites/rbd/migration`.